File: src/main/java/com/dh/ondot/notification/domain/repository/SubwayAlertRepository.java
package com.dh.ondot.notification.domain.repository;

import com.dh.ondot.notification.domain.SubwayAlert;
import org.springframework.data.jpa.repository.JpaRepository;

import java.time.LocalDateTime;
import java.util.List;

public interface SubwayAlertRepository extends JpaRepository<SubwayAlert,Long> {
    List<SubwayAlert> findAllByCreatedAtBetween(LocalDateTime start, LocalDateTime end);
}

File: src/main/kotlin/com/dh/ondot/notification/domain/repository/SubwayAlertRepository.kt
package com.dh.ondot.notification.domain.repository

import com.dh.ondot.notification.domain.SubwayAlert
import org.springframework.data.jpa.repository.JpaRepository
import java.time.Instant

interface SubwayAlertRepository : JpaRepository<SubwayAlert, Long> {
    fun findAllByCreatedAtBetween(start: Instant, end: Instant): List<SubwayAlert>
}
